src/test/java/helpers/UserCreator.java

```diff
@@ -0,0 +1,20 @@
+package helpers;
+
+import model.User;
+
+public class UserCreator {
+
+    public static final String TESTDATA_USER_NAME = "standard_user";
```

Comment:
please remove Test Data to separate environment-dependent files and ensure execution in different envs

Reply:
+/-